apps/remix-ide/src/app.js

```diff
@@ -50,8 +51,16 @@ const Editor = require('./app/editor/editor')
 const Terminal = require('./app/panels/terminal')
 const ContextualListener = require('./app/editor/contextualListener')
 
-class AppComponent {
+const profile = {
+  name: 'app',
+  displayName: 'App',
+  description: 'Application',
+  methods: []
+}
+
+class AppComponent extends Plugin {
```

Comment:
cool, this need to be added to the engine and activated too.
cc @bunsenstraat cause that might interfer with what he's doing currently